apiserver/facades/client/bundle/state.go

```diff
@@ -12,6 +12,7 @@ import (
 type Backend interface {
 	ExportPartial(cfg state.ExportConfig) (description.Model, error)
 	GetExportConfig() state.ExportConfig
+	Machine(id string) (*state.Machine, error)
```

Comment:
The result needs to be a Machine interface not a state.Machine.
The interface has just the method on it we use in the code and which we then add to the mock